src/utils/require-template.js

```diff
@@ -19,30 +19,34 @@ function isValidTemplate(template){
   ].every((method) => template[method]);
 }
 
+const getOcTemplate = (path) => {
+  if (require.cache && !!require.cache[path]) {
+    delete require.cache[path];
+  }
+  return require(path);
+};
 
-module.exports = function(template) {
+module.exports = function (template) {
   let ocTemplate;
   const localTemplate = path.join(__dirname, '../../', 'node_modules', template);
   const relativeTemplate = path.resolve('.', 'node_modules', template);
 
   try {
-    if (require.cache && !!require.cache[localTemplate]) {
-      delete require.cache[localTemplate];
-    }
-    ocTemplate = require(localTemplate);
-  } catch(err) {
+    ocTemplate = getOcTemplate(template);
```

Comment:
Yeah, I was doing it now... and then I face with this...

![imatge](https://cloud.githubusercontent.com/assets/153305/26409082/d7096a0a-409f-11e7-9a49-97fb76090cf4.png)

After seeing the error... I thought that maybe your changes now could allow me to publish from the widget folder itself, so I tried:

![imatge](https://cloud.githubusercontent.com/assets/153305/26409125/f27254fa-409f-11e7-9e9a-0665b9fb333f.png)


the neverending story... 😅